runtime/frequency/src/lib.rs

```diff
@@ -228,14 +228,20 @@ pub type Executive = frame_executive::Executive<
 /// to even the core data structures.
 pub mod opaque {
 	use super::*;
+	use sp_runtime::{
```

Comment:
Non-blocking: Do we have a convention for `use` statements in files and modules?  It seems kind of arbitrary.

Reply:
This follows templates Polkadot template. We tried to match it as much as possible.